### PR TITLE
fix: flaws in web/api/h* batch 1

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/web/api/hmacimportparams/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h2 id="Properties">Properties</h2>
  <dd>A {{domxref("DOMString")}}. This should be set to <code>HMAC</code>.</dd>
  <dt><code>hash</code></dt>
  <dd>
- <p>A {{domxref("DOMString")}} representing the name of the <a href="/en-US/docs/Web/API/Web_Crypto_API/Supported_algorithms#Digest_algorithms">digest function</a> to use. The can take a value of <code>SHA-256</code>, <code>SHA-384</code>, or <code>SHA-512</code>.</p>
+ <p>A {{domxref("DOMString")}} representing the name of the <a href="/en-US/docs/Web/API/SubtleCrypto#digest_algorithms">digest function</a> to use. The can take a value of <code>SHA-256</code>, <code>SHA-384</code>, or <code>SHA-512</code>.</p>
```
Comment (wbamberg) [inline files/en-us/web/api/hmacimportparams/index.html:22]: This should be /Web/API/SubtleCrypto/digest#supported_algorithms .

File: files/en-us/web/api/hmackeygenparams/index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@ <h2 id="Properties">Properties</h2>
  <dd>A {{domxref("DOMString")}}. This should be set to <code>HMAC</code>.</dd>
  <dt><code>hash</code></dt>
  <dd>
- <p>A {{domxref("DOMString")}} representing the name of the <a href="/en-US/docs/Web/API/Web_Crypto_API/Supported_algorithms#Digest_algorithms">digest function</a> to use. You can pass any of <code>SHA-1</code>, <code>SHA-256</code>, <code>SHA-384</code>, or <code>SHA-512</code> here.</p>
+ <p>A {{domxref("DOMString")}} representing the name of the <a href="/en-US/docs/Web/API/SubtleCrypto#digest_algorithms">digest function</a> to use. You can pass any of <code>SHA-1</code>, <code>SHA-256</code>, <code>SHA-384</code>, or <code>SHA-512</code> here.</p>
```
Comment (wbamberg) [inline files/en-us/web/api/hmackeygenparams/index.html:22]: This should be /Web/API/SubtleCrypto/digest#supported_algorithms .

**Review by wbamberg — COMMENTED** [review]
This looks fine, although there are a few places where the original link was wrong AFAICT. I don't know if it's worth fixing these, while we are in there.

File: files/en-us/web/api/hmackeygenparams/index.html (hunk 2 above)
Comment (nschonni) [reply files/en-us/web/api/hmackeygenparams/index.html:22]:
```suggestion
 <p>A {{domxref("DOMString")}} representing the name of the <a href="/en-US/docs/Web/API/SubtleCrypto/digest#supported_algorithms">digest function</a> to use. You can pass any of <code>SHA-1</code>, <code>SHA-256</code>, <code>SHA-384</code>, or <code>SHA-512</code> here.</p>
```

File: files/en-us/web/api/hmacimportparams/index.html (hunk 1 above)
Comment (nschonni) [reply files/en-us/web/api/hmacimportparams/index.html:22]:
```suggestion
 <p>A {{domxref("DOMString")}} representing the name of the <a href="/en-US/docs//Web/API/SubtleCrypto/digest#supported_algorithms">digest function</a> to use. The can take a value of <code>SHA-256</code>, <code>SHA-384</code>, or <code>SHA-512</code>.</p>
```

File: files/en-us/web/api/hashchangeevent/index.html [hunk 3]
```diff
@@ -65,7 +65,7 @@ <h3 id="Basic_example">Basic example</h3>
 
 <h2 id="Polyfill">Polyfill</h2>
 
-<p>There are several fallback scripts listed on the <a href="https://github.com/Modernizr/Modernizr/wiki/HTML5-Cross-Browser-Polyfills">Modernizr GitHub page</a>. Basically, those scripts check the {{domxref("HTMLHyperlinkElementUtils.hash", "location.hash")}} at a regular interval. Here is a version that allows only one handler to be bound to the {{domxref("WindowEventHandlers.onhashchange", "onhashchange")}} property:</p>
+<p>There are several fallback scripts listed on the <a href="https://github.com/Modernizr/Modernizr/wiki/HTML5-Cross-Browser-Polyfills">Modernizr GitHub page</a>. Basically, those scripts check the {{domxref("HTMLAnchorElement/hash", "location.hash")}} at a regular interval. Here is a version that allows only one handler to be bound to the {{domxref("WindowEventHandlers.onhashchange", "onhashchange")}} property:</p>
```
Comment (wbamberg) [inline files/en-us/web/api/hashchangeevent/index.html:68]:
```suggestion
<p>There are several fallback scripts listed on the <a href="https://github.com/Modernizr/Modernizr/wiki/HTML5-Cross-Browser-Polyfills">Modernizr GitHub page</a>. Basically, those scripts check the <code>hash</code> property of {{domxref("Window.location")}} at a regular interval. Here is a version that allows only one handler to be bound to the {{domxref("WindowEventHandlers.onhashchange", "onhashchange")}} property:</p>
```

**Review by wbamberg — APPROVED** [review]
Thanks @nschonni !

**Pull request merged**